Definitions/RISCVnnn.json

```diff
@@ -0,0 +1,353 @@
+//
+// Copyright (c) 2022 Saleem Abdulrasool <compnerd@compnerd.org>
+// All Rights Reserved.
+//
+
+{
+    "namespace": "RISCV@WORDSIZE@",
+    "register-sets": {
+        "gpr": {
+            "*": {
+                "bit-size": @WORDSIZE@,
+                "encoding": "int",
+                "format": "hex",
+
+                "base-gdb-reg-number": 0,
+                "base-ehframe-reg-number": 0,
+                "base-dwarf-reg-number": 0
+            },
+
+            "x0": {
+                "alternate-name": "zero",
```

Comment:
Nit: For the alternate-names, it would be useful to have a link to a reference where these are defined.